client/x/evmstaking/keeper/withdraw.go

```diff
@@ -244,6 +244,16 @@ func (k Keeper) EnqueueEligiblePartialWithdrawal(ctx context.Context, withdrawal
 }
 
 func (k Keeper) ProcessWithdraw(ctx context.Context, ev *bindings.IPTokenStakingWithdraw) error {
+	isInSingularity, err := k.IsSingularity(ctx)
+	if err != nil {
+		return errors.Wrap(err, "check if it is singularity")
+	}
+
+	if isInSingularity {
+		log.Info(ctx, "Withdraw event detected, but it is not processed since current block is singularity")
```

Comment:
change to debug?

Reply:
changed in [2150314](https://github.com/piplabs/story/pull/202/commits/2150314811ef720fd1cf5ed4bb9e5ce7e5614da3)